backend/src/models/User.js

```diff
@@ -80,9 +80,19 @@ module.exports = {
   notifications: {
     type: 'relationship',
     relationship: 'NOTIFIED',
-    target: 'Notification',
+    target: 'User',
     direction: 'in',
   },
+  termsAndConditionsAgreedVersion: {
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-08-30T09:11:20Z" title="Friday, August 30th 2019, 11:11:20 am +02:00">Aug 30, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -83,4 +83,14 @@ module.exports = {
     target: 'Notification',
     direction: 'in',
   },
+  termsAndConditionsAgreedVersion: {
+    type: 'string',
+    allow: [null],
+  },
+  termsAndConditionsAgreedAt: {
```


Isn’t this used at the moment? Means especially not written to the database.
I don’t think so! 😅

- Write it to the database.
- Or search for it in the codebase and remove it all over the place …